core/chains/evm/chain.go

```diff
@@ -178,13 +181,19 @@ type RelayerConfig struct {
 	GenGasEstimator   func(*big.Int) gas.EvmFeeEstimator
 }
 
+func (r *RelayerConfig) EVMConfigs() evmtypes.Configs {
+	if r.operationalConfigs == nil {
+		r.operationalConfigs = chains.NewConfigs[utils.Big, evmtypes.Node](r.AppConfig.EVMConfigs())
+	}
+	return r.operationalConfigs
```

Comment:
Does this need to be safe for concurrent use? e.g. sync.Once?

Reply:
i don't think so because it is idempotent since the value of AppConfig.EVM is constant. However i guess it could leak a resource.

changed it and added a test